reference/forms/types/dateinterval.rst

```diff
@@ -0,0 +1,290 @@
+ISO 8601 duration string you'd need to set ``input`` to ``string``:
+
+.. code-block:: php
+
+    $builder->add('remindEvery', DateIntervalType::class, array(
+        'input'  => 'string',
+        'widget' => 'choice',
+    ));
+
+The field also supports an ``array`` as valid ``input`` option values.
+
+Field Options
+-------------
+
+days
+~~~~
+
+**type**: ``array`` **default**: 0 to 31
+
+List of days available to the days field type. This option is only relevant
+when the ``widget`` option is set to ``choice``::
+
+    'days' => range(1, 31)
+
+placeholder
+~~~~~~~~~~~
+
+**type**: ``string`` or ``array``
+
+If your widget option is set to ``choice``, then this field will be represented
+as a series of ``select`` boxes. The ``placeholder`` option can be used to
+add a "blank" entry to the top of each select box::
+
+    $builder->add('remindEvery', DateIntervalType::class, array(
+        'placeholder' => '',
+    ));
+
+Alternatively, you can specify a string to be displayed for the "blank" value::
+
+    $builder->add('remindEvery', DateIntervalType::class, array(
+        'placeholder' => array('years' => 'Years', 'months' => 'Months', 'days' => 'Days')
+    ));
+
+hours
+~~~~~
+
+**type**: ``array`` **default**: 0 to 24
+
+List of hours available to the hours field type. This option is only relevant
+when the ``widget`` option is set to ``choice``::
+
+    'hours' => range(1, 24)
+
+input
+~~~~~
+
+**type**: ``string`` **default**: ``dateinterval``
+
+The format of the *input* data - i.e. the format that the interval is stored on
+your underlying object. Valid values are:
+
+* ``string`` (a string formatted with `RFC 3339`_ standard, e.g. ``P7Y6M5DT12H15M30S``)
+* ``dateinterval`` (a ``DateInterval`` object)
+* ``array`` (e.g. ``array('days' => '1', 'hours' => '12',)``)
+
+The value that comes back from the form will also be normalized back into
+this format.
+
+minutes
+~~~~~~~
+
+**type**: ``array`` **default**: 0 to 60
+
+List of minutes available to the minutes field type. This option is only relevant
+when the ``widget`` option is set to ``choice``::
+
+    'minutes' => range(1, 60)
+
+months
+~~~~~~
+
+**type**: ``array`` **default**: 0 to 12
+
+List of months available to the months field type. This option is only relevant
+when the ``widget`` option is set to ``choice``::
+
+    'months' => range(1, 12)
+
+seconds
+~~~~~~~
+
+**type**: ``array`` **default**: 0 to 60
+
+List of seconds available to the seconds field type. This option is only relevant
+when the ``widget`` option is set to ``choice``::
+
+    'seconds' => range(1, 60)
+
+weeks
+~~~~~
+
+**type**: ``array`` **default**: 0 to 52
+
+List of weeks available to the weeks field type. This option is only relevant
+when the ``widget`` option is set to ``choice``::
+
+    'weeks' => range(1, 52)
+
+widget
+~~~~~~
+
+**type**: ``string`` **default**: ``choice``
+
+Defines the ``widget`` option for the single interval component inputs.
```

Comment:
I miss the list of possible values for this option. Is just `choice` and `text`?